### DISCUSSION TITLE
Allow font weight specification in `style` - disallow conflict

### DISCUSSION THREAD

**Discussion by Arpita-Jaiswal** [fastn-stack/fastn#327] [Ideas & RFCs]
In `ftd.text`, `style` property only contains `italic`, `underline` and `strike`. Let's also add font weight.

Instead of font weight as an integer, we will use keywords that [google uses](https://fonts.google.com/specimen/Roboto).

We should also disallow conflicting `style: underline regular bold`, `regular` and `bold` are conflicting specification.

**Comment by amitu** [comment]
Bulk of this is implemented in https://github.com/FifthTry/ftd/pull/329. The conflicting check is pending.
